department-of-veterans-affairs/next-build
# 14574 implement header footer

## Description
Adds header and footer to next-build
Renders nav components and mega menu.  Not all content-build functionality is ported over.
Data is sourced from a JSON object in importHeaderData.js.  This data was pulled from the prod JSON file

Relates to #14574
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14574

## Testing done
Local visual testing

## Screenshots
<img width="1531" alt="Screen Shot 2023-08-14 at 10 28 46 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/ed6be919-c9e4-42cf-aea7-b21ccfe46733">
<img width="1531" alt="Screen Shot 2023-08-14 at 10 28 37 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/b9827a8b-1691-4323-91a5-07c6fd537c36">
<img width="1531" alt="Screen Shot 2023-08-14 at 10 30 54 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/61624970/15b97e7e-9b83-4497-9b52-d5419d962646">

## QA steps

What needs to be checked to prove this works?  
Navigate to any existing page in next build and verify that the header and footer are rendered

What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?

As user navigating va.gov

1. Do this
    - [ ] Validate that the header renders
2. Then
    - [ ] Validate that the footer renders
3. Then validate Acceptance Criteria from issue
    - [ ]  Header and footer menus render when a Next Build static page is viewed
    - [ ] Data is accurate and sourced from a current headerFooter.json and written to page
  
### Definition of Done


### Is this PR blocked by another PR?
